newIDE/app/public/external/piskel/piskel-main.js

```diff
@@ -365,31 +364,18 @@ ipcRenderer.on('piskel-load-animation', (event, receivedOptions) => {
 
   // Load a custom save file(s) header
   const pathEditorHeaderDiv = document.getElementById('path-editor-header');
-  const headerStyle = {
```

Comment:
Removing these hardcoded styles is definitely a nice thing 👍

Reply:
They were an eye sore 😔 sorry I ever put them there. Was a pain in the neck to check the styling and reuse it